templates/CRM/Contribute/Form/Contribution/Confirm.tpl

```diff
@@ -111,10 +111,30 @@
             {else}
               {if $frequency_interval > 1}
                 <p>
```

Comment:
i.e. do we need to remove this p tag (and the associated closing tag)?

Reply:
I did a little run and it does seem to create an extra `<p>`, which looks like it creates some visual extra blank space.

And also there's a typo - the first group all say "every every".